src/Microsoft.DotNet.Darc/DarcLib/Models/Darc/DownloadedAsset.cs

```diff
@@ -18,17 +19,25 @@ public class DownloadedAsset
     /// <summary>
     /// Target location where the asset was downloaded to for the release style layout
     /// </summary>
-    public string ReleaseLayoutTargetLocation { get; set; }
+    public string SeparatedLayoutTargetLocation { get; set; }
     /// <summary>
     /// Target location where the asset was downloaded to for the unified-style (sans repo/build #) layout
     /// </summary>
     public string UnifiedLayoutTargetLocation { get; set; }
     /// <summary>
     /// True if the asset download was successful. If false, Asset is the only valid property
     /// </summary>
+    public string ReleasePackageLayoutTargetLocation { get; set; }
     public bool Successful { get; set; }
     /// <summary>
     /// Location type of the asset that actually got downloaded.
     /// </summary>
     public LocationType LocationType { get; set; }
+    /// <summary>
+    /// Size of the artifact in bytes
+    /// </summary>
+    public long SizeInBytes { get; set; }
+    public string FileHash { get; set; }
```

Comment:
I don't see where these new properties are being set. Am I missing something?

Reply:
In GatherDropOperation.cs

```
private async Task AccountForAssetAsync(DownloadedAsset downloadedAsset)
    {
        // First record the file size in bytes and file hash.
        // Then examine the file type and unpack the file if we support the type.
        FileInfo fileInfo = new FileInfo(downloadedAsset.UnifiedLayoutTargetLocation);
        downloadedAsset.SizeInBytes = fileInfo.Length;
        using (Stream reader = File.OpenRead(downloadedAsset.UnifiedLayoutTargetLocation))
        using (SHA256 fileHasher = SHA256.Create())
        {
            downloadedAsset.FileHash = Convert.ToBase64String(await fileHasher.ComputeHashAsync(reader));
        }
```